### PR TITLE
added single product page

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added single-product description page.

- details about the product
- the price of the product
- ratings for the product
- talk to seller button

proddesc

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
Code looks clean and understandable with the use of CSS variables.
Good to merge.

File: productPage/singleProduct.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+.product-details {
+    align-items: center;
+    display: flex;
+    flex-flow: row wrap;
+}
+
+.product-details img {
+    width: 50%;
+    height: 80%;
```
Comment (lannisterrr) [inline productPage/singleProduct.css:17]: We should not use height for images ideally.

Comment (GuptaEesh) [reply productPage/singleProduct.css:17]: Yeah just to restrict it to the same view-page I have done that.

**Review by lannisterrr — COMMENTED** [review]
code seems fine to merge.

**Comment by GuptaEesh** [comment]
2 reviews done. Merging the branch now.

**Pull request merged**